tests/snuba/api/endpoints/test_organization_events_trace.py

```diff
@@ -7,16 +8,18 @@
 from sentry.testutils.helpers.datetime import before_now
 
 
-class OrganizationEventsTrendsEndpointBase(APITestCase, SnubaTestCase):
+class OrganizationEventsTraceEndpointBase(APITestCase, SnubaTestCase):
     FEATURES = ["organizations:trace-view-quick", "organizations:global-views"]
 
-    def create_event(self, trace, transaction, spans, parent_span_id, project_id):
+    def create_event(self, trace, transaction, spans, parent_span_id, project_id, duration=4000):
+        start = before_now(milliseconds=duration + 60 * 1000)
```

Comment:
This seems a little cleaner

```suggestion
        start = before_now(minutes=1, milliseconds=duration)
```